### PR TITLE
pop up before deleting an appointment + README file updates

### PR DESCRIPTION
**Pull request by ritabiniash**
pop up : 
image

readme: 

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,8 +66,26 @@ request.
...
-4. To start the Coordinator app:
+5. To start the Coordinator app:
    `yarn run start-coordinator`
-5. ENJOY!
+
+## Before merging
+
+To manage our development environment some automatic tests 
+are triggered with each pull request. 
+Run this steps locally to make sure they will pass:
```
Comment (yaron1m) [inline README.md:80]: *these

File: packages/coordinator/src/components/AppointmentPreview/AppointmentPreview.tsx [hunk 2]
```diff
@@ -142,3 +150,23 @@ function DeleteAppointmentButton(props: {
...
+  closePopup: () => void;
+  onDeleteClick: () => void;
+}) {
+  const { openPopup, closePopup, onDeleteClick } = props;
+
+  return (
+    <Popup
+      open={openPopup}
+      onClose={closePopup}
+      content={"האם ברצונך למחוק תור זה ?"}
```
Comment (yaron1m) [inline packages/coordinator/src/components/AppointmentPreview/AppointmentPreview.tsx:165]: Please remove the space before the `?`

**Review by yaron1m — APPROVED** [review]
Nice work!

File: README.md (hunk 1 above)
Comment (ritabiniash) [reply README.md:80]: shame

**Comment by yaron1m** [comment]
Can we merge this?

**Pull request merged**